docs/tool/js/views/map-view.js

```diff
@@ -994,5 +995,27 @@
                     'filter': ['==', 'nlihc_id', data]
                 });
             }
+        },
+        zoomToFilteredProjects: function(msg, data){
+            if (getState().filteredProjectsAvailable.length < 2 ) {
```

Comment:
I wanted to avoid the "Reset zoom" button coming up on load, but, yeah, if we set the original zoom and center to match the fitBounds of all projects, we could remove it.